### PR TITLE
Added connected workspace reference

### PR DESCRIPTION
**Pull request by scott-epperly**
Recent SR made it clear that the feature disparity for renaming a dedicated SQL pool in a Synapse workspace is not clearly documented.  Renaming cannot occur under a standard workspace (where the workspace is created first and then the dedicated SQL pool is created), but is supported in a connected workspace scenario.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@WilliamDAssafMSFT - Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/t-sql/statements/alter-database-transact-sql.md [hunk 1]
```diff
@@ -1044,8 +1044,8 @@ Renames the database with the name specified as *new_database_name*.
  
 The 'MODIFY NAME' option has some support limitations in Azure Synapse:
  - Unsupported with Azure Synapse serverless pools
- - Unsupported with dedicated SQL pools created in your Azure Synapse Workspace
- - Supported with dedicated SQL pools (formerly SQL DW) created via the [Azure portal](https://portal.azure.com/#create/Microsoft.SQLDataWarehouse)
+ - Unsupported with SQL dedicated pools created in your Azure Synapse Workspace
+ - Supported with dedicated SQL pools (formerly SQL DW) created via the [Azure portal](https://portal.azure.com/#create/Microsoft.SQLDataWarehouse), including those with a [connected workspace](https://docs.microsoft.com/azure/synapse-analytics/sql-data-warehouse/workspace-connected-create)
```
Comment (scott-epperly) [inline docs/t-sql/statements/alter-database-transact-sql.md:1048]:
```suggestion
 - Supported with dedicated SQL pools (formerly SQL DW) created via the [Azure portal](https://portal.azure.com/#create/Microsoft.SQLDataWarehouse), including those with a [connected workspace](/azure/synapse-analytics/sql-data-warehouse/workspace-connected-create)

```

File: docs/t-sql/statements/alter-database-transact-sql.md [hunk 2]
```diff
@@ -1044,8 +1044,8 @@ Renames the database with the name specified as *new_database_name*.
  
 The 'MODIFY NAME' option has some support limitations in Azure Synapse:
  - Unsupported with Azure Synapse serverless pools
- - Unsupported with dedicated SQL pools created in your Azure Synapse Workspace
- - Supported with dedicated SQL pools (formerly SQL DW) created via the [Azure portal](https://portal.azure.com/#create/Microsoft.SQLDataWarehouse)
+ - Unsupported with SQL dedicated pools created in your Azure Synapse Workspace
```
Comment (scott-epperly) [inline docs/t-sql/statements/alter-database-transact-sql.md:1047]:
```suggestion
 - Unsupported with dedicated SQL pools created in your Azure Synapse Workspace

```

**Review by scott-epperly — COMMENTED** [review]
here's what it should look like

File: docs/t-sql/statements/alter-database-transact-sql.md [hunk 3]
```diff
@@ -1045,7 +1045,9 @@ Renames the database with the name specified as *new_database_name*.
 The 'MODIFY NAME' option has some support limitations in Azure Synapse:
  - Unsupported with Azure Synapse serverless pools
  - Unsupported with dedicated SQL pools created in your Azure Synapse Workspace
- - Supported with dedicated SQL pools (formerly SQL DW) created via the [Azure portal](https://portal.azure.com/#create/Microsoft.SQLDataWarehouse)
+
```
Comment (scott-epperly) [inline docs/t-sql/statements/alter-database-transact-sql.md:1048]:
```suggestion
```

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Pull request merged**